Taskfile.yml

```diff
@@ -102,7 +102,7 @@ tasks:
     cmds:
       - task: bash:build
         vars:
-          PLATFORM: '{{if eq .PLATFORM "all"}}{{.SUPPORTED_PLATFORMS}}{{else if .PLATFORM}}{{.PLATFORM}}{{else}}{{.LOCAL_PLATFORM}}{{end}}'
+          PLATFORM: '{{.PLATFORM | default .LOCAL_PLATFORM}}'
```

Comment:
Made the base `Taskfile.yml` a little more simple.  The downside to this approach is that the passing of the `SUPPORTED_PLATFORMS` var is a little less clear because it's in set the including `Taskfile` and only referenced in the included `Taskfile` without being explicitly passed.  But, that's how Taskfiles work /shrug